ga4gh/datamodel/variants.py

```diff
@@ -335,9 +353,8 @@ def _addFile(self, filename):
                 self._chromFileMap[chrom] = varFile
 
     def _convertGaCall(self, recordId, name, pysamCall, genotypeData):
-        call = protocol.GACall()
-        call.callSetId = recordId
-        call.callSetName = name
+        callSet = self.getCallSet(name)
+        call = callSet.getGaCall()
```

Comment:
so, replace this line with:
```
call = protocol.GACall()
call.callSetId = callSet._id
call.callSetName = callSet._sampleName
gaCall.sampleId = callSet._sampleName
```
And just to check - should both the above always be set to the same thing? If so, why do we have two separate fields in the protocol definition?